super/Login_db/bot.py

```
@@ -69,8 +69,32 @@ def jls_extract_def():
 
 main_args, credentials = jls_extract_def()
 
-def log_one(site, user, result, action=""):
+userinfo = 0
```

Pylint (3.3.7)

[convention] Constant name "userinfo" doesn't conform to UPPER_CASE naming style

(C0103)